modules/core/src/loaders/playable-loader-interface.js

```diff
@@ -31,36 +31,46 @@ export default class PlayableLoaderInterface extends LoaderInterface {
    * @param {Number} timestamp
    */
   seek(timestamp) {
+    this.set('timestamp', timestamp);
     throw new Error('Not implemented');
```

Comment:
Remove the throw if we are providing an implementation.  But this seems curious.

We appear to be setting an expectation for what keys & state are stored based on these methods. But an implementation may *not* want to seek (say if the timestamp is out of the bounds).  *But* it still expects that the key 'timestamp' is used.

Is there a hard reason to force the keys here?

Reply:
i will turn this into a comment. i remember being tripped up when i created a loader implementation because i wasn't setting the timestamp